examples/rollbar-reactive-streams-reactor/src/main/java/com/rollbar/reactivestreams/reactor/example/Application.java

```diff
@@ -0,0 +1,57 @@
+package com.rollbar.reactivestreams.reactor.example;
+
+import com.rollbar.reactivestreams.notifier.ReactorRollbar;
+import com.rollbar.reactivestreams.notifier.config.Config;
+import com.rollbar.reactivestreams.notifier.config.ConfigBuilder;
+import com.rollbar.reactivestreams.notifier.sender.http.ReactorAsyncHttpClient;
+import reactor.core.publisher.Hooks;
+import reactor.core.publisher.Mono;
+import reactor.netty.DisposableChannel;
+import reactor.netty.DisposableServer;
+import reactor.netty.http.client.HttpClient;
+import reactor.netty.http.client.HttpClientResponse;
+import reactor.netty.tcp.TcpServer;
+
+/**
+ * Application example using rollbar-reactive-streams-reactor.
+ */
+public class Application {
+  /**
+   * Main method.
+   * @param args the args.
+   */
+  public static void main(String[] args) throws Exception {
+    Hooks.onOperatorDebug();
+
+    Config config = ConfigBuilder
+        .withAccessToken(System.getenv("ROLLBAR_ACCESSTOKEN"))
+        .httpClient(new ReactorAsyncHttpClient.Builder().build())
+        .environment("development")
+        .build();
+
+    ReactorRollbar rollbar = new ReactorRollbar(config);
+
+    // We'll create a misbehaving HTTP server that immediately disconnects clients
+    DisposableServer badServer = TcpServer.create().port(0)
+        .doOnConnection(DisposableChannel::disposeNow)
+        .bindNow();
+
+    String url = "http://localhost:" + badServer.port();
+
+    // Rollbar.logMonoError will log the error and return the original Mono
+    Mono<HttpClientResponse> request = HttpClient.create()
+        .get()
+        .uri(url)
+        .response()
+        .onErrorResume(rollbar::logMonoError);
+
+    // The Mono still contains the error, so we'll replace it with an empty Mono and wait for it to
+    // complete.
+    request.onErrorResume(r -> Mono.empty()).block();
```

Comment:
Our reactive notifier right now only provides the `catch -> report -> re-throw` workflow, and since this is just an example application, we clear the error after the notifier has done its part, to let the app finish successfully. 
We could simply not clear the error, but then we'd get an application crash. 

I considered adding the `catch -> report -> forget` option but even if we ignore what we think of it as a practice, it's not a great approach because to swallow the error we'd have to make the mono empty. This is technically valid, but users will assume it has a value and might attach handlers to `onNext` or `onError` expecting one of the two to be called (eg. if your mono represents the result of a HTTP operation, you expect a result or an error, not an mono that's been emptied by the Rollbar SDK.)

I'll add an explanation that the user at that point is free to deal with the original error how they see fit, and we're just clearing it to let the application finish successfully.